tdvt/tdvt/tdvt_core.py

```diff
@@ -475,6 +490,8 @@ def get_csv_row_data(tds_name, test_name, test_path, test_result, test_case_inde
     if not test_result or not test_result.get_test_case_count() or not test_result.get_test_case(test_case_index):
         error_msg= test_result.get_failure_message() if test_result else None
         error_type= test_result.get_failure_message() if test_result else None
+        if isinstance(test_result.error_status, TestErrorExpected):
```

Comment:
I think this should go into the TestResult class in test_results.py. You could probably update all_passed there and use that.